### PR TITLE
Add input toggling and fix output toggling

### PR DESCRIPTION
**Pull request by captainsafia**
TOGGLE ALL THE THINGS.

### CODE REVIEW

**Comment by captainsafia** [comment]
YAY! Ready for merge.

output-input-hiding

**Comment by captainsafia** [comment]
One another note, someone who actually knows how to make things look pretty should take a stab at prettying up the context menu.

File: test/renderer/actions/index_spec.js [hunk 1]
```diff
@@ -366,14 +366,23 @@ describe('copyCell', () => {
...
+  it('creates a CHANGE_OUTPUT_VISIBILITY action', () => {
     expect(actions.changeOutputVisibility('235')).to.deep.equal({
       type: constants.CHANGE_OUTPUT_VISIBILITY,
       id: '235',
     });
   });
 });
 
+describe('changeInputVisibility', () => {
+  it('creates a CHANGE_INPUT_VISIBILITY', () => {
```
Comment (jdetle) [inline test/renderer/actions/index_spec.js]: This is an action too, no?

Comment (captainsafia) [inline test/renderer/actions/index_spec.js]: Bleh! Yeah. I'll fix that. Let me know if there is anything else that I need to fix and I'll do it in one commit.

**Comment by jdetle** [comment]
I dont see anything else of note.

**Comment by captainsafia** [comment]
OK. This should be good to merge.

**Pull request merged**